configure

```diff
@@ -542,7 +546,14 @@ def is_arm_hard_float_abi():
 def host_arch_cc():
   """Host architecture check using the CC command."""
 
-  k = cc_macros()
+  if sys.platform.startswith('aix'):
+    k = cc_macros('gcc')
+    matchup = {
+      '_ARCH_PPC'   : 'ppc',
+      '_ARCH_PPC64' : 'ppc64',
+    }
```

Comment:
This is a no-op, isn't it?  The second definition of `matchup` a few lines below overrides it again.